perfkitbenchmarker/virtual_machine.py

```diff
@@ -285,36 +314,65 @@ def RemoteCopy(self, file_path, remote_path='', copy_to=True):
                     (retcode, full_cmd, stdout, stderr))
       raise errors.VmUtil.SshConnectionError(error_text)
 
+  def _PushLongRunningCommandScripts(self):
```

Comment:
Can this function be called to push script for any purpose (in addition to pushing it only for long running commands)? I don't see anything specific here in this function wrt long running commands (but I could be wrong here), please consider renaming this function and make it generic so anyone can use it to push a script (safely) to a VM.